templates/pages/admin/entity/survey_config.html.twig

```diff
@@ -0,0 +1,136 @@
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{{ fields.dropdownNumberField('inquest_delay' ~ config_suffix, inquest_delay, __('Create survey after'), {
+    full_width: true,
+    min: 1,
+    max: 90,
+    unit: 'day',
+    toadd: {
+        0: __('As soon as possible'),
+    }
+}) }}
+
+{{ fields.dropdownNumberField('inquest_rate' ~ config_suffix, inquest_rate, __('Rate to trigger survey'), {
+    full_width: true,
+    min: 0,
+    max: 100,
+    step: 10,
+    unit: '%',
+    toadd: {
+        0: __('Disabled')
+    }
+}) }}
+
+{{ fields.dropdownNumberField('inquest_duration' ~ config_suffix, inquest_duration, __('Duration of survey'), {
+    full_width: true,
+    min: 1,
+    max: 100,
+    unit: 'day',
+    toadd: {
+        0: __('Unspecified')
+    }
+}) }}
+
+{{ fields.dropdownNumberField('inquest_max_rate' ~ config_suffix, inquest_max_rate, __('Max rate'), {
+    full_width: true,
+    min: 1,
+    max: 10,
+}) }}
+
+{{ fields.numberField('inquest_default_rate' ~ config_suffix, inquest_default_rate, __('Default rate'), {
+    full_width: true,
+    min: 1,
+    max: 10,
+}) }}
+
+{{ fields.numberField('inquest_mandatory_comment' ~ config_suffix, inquest_mandatory_comment, __('Comment required if score is <= to'), {
+    full_width: true,
+    min: 1,
+    max: 10,
+    toadd: {
+        0: __('Disabled')
+    }
+}) }}
+
+{{ fields.datetimeField('max_closedate' ~ config_suffix, max_closedate, __('For %s closed after')|format(itemtype|itemtype_name(get_plural_number())), {
+    full_width: true,
+    maybeempty: true,
+    timestep: 1
+}) }}
+
+{% if inquest_config == constant('CommonITILSatisfaction::TYPE_EXTERNAL') %}
+    {% set tag_prefix = itemtype|upper %}
+    {% set ticket_only_tags = " [REQUESTTYPE_ID] [REQUESTTYPE_NAME] [TICKETTYPE_NAME] [TICKETTYPE_ID] [SLA_TTO_ID] [SLA_TTO_NAME] [SLA_TTR_ID] [SLA_TTR_NAME] [SLALEVEL_ID] [SLALEVEL_NAME]" %}
+    {% set tags = [
+        '[' ~ tag_prefix ~ '_ID]', '[' ~ tag_prefix ~ '_NAME]', '[' ~ tag_prefix ~ '_CREATEDATE]', '[' ~ tag_prefix ~ '_SOLVEDATE]',
+        '[' ~ tag_prefix ~ '_PRIORITY]', '[' ~ tag_prefix ~ '_PRIORITYNAME]', '[ITILCATEGORY_ID]', '[ITILCATEGORY_NAME]',
+        '[SOLUTIONTYPE_ID]', '[SOLUTIONTYPE_NAME]'
+    ]|join(' ') ~ (itemtype == 'Ticket' ? ticket_only_tags : '') %}
+    {{ fields.htmlField('', tags|e, __('Valid tags'), {
+        full_width: true
+    }) }}
+    {{ fields.textField('inquest_URL' ~ config_suffix, inquest_URL, __('URL'), {
+        full_width: true,
+    }) }}
```

Comment:
When switching to external survey, this field should be shown, but is not.
![image](https://github.com/glpi-project/glpi/assets/33253653/03a158dc-ecc0-424a-8453-4488d3f4f5e0)